os/board/rtl8721csm/src/component/soc/realtek/amebad/fwlib/ram_hp/rtl8721dhp_app_start.c

```diff
@@ -1047,6 +1047,14 @@ void app_mpu_s_nocache_init(void)
 #endif
 }
 
+#ifdef CONFIG_AMEBAD_TRUSTZONE
+void app_hardfualt_s_prehanlder(uint32_t fault_id)
+{
+	//write reboot reason, TrustZone watchdog
+	BKUP_Write(BKUP_REG1, 253);
```

Comment:
It is better to use `REBOOT_SYSTEM_TZWD_RESET`.

Reply:
Thank you for reminder! I changed as asked.